tests/src/admin/tests/utils/mod.rs

```diff
@@ -0,0 +1,143 @@
+use fuels::{
+    prelude::{
+        abigen, launch_custom_provider_and_get_wallets, Contract, LoadConfiguration,
+        StorageConfiguration, TxParameters, WalletUnlocked, WalletsConfig,
+    },
+    programs::call_response::FuelCallResponse,
+    types::Identity,
+};
+
+// Load abi from json
+abigen!(Contract(
+    name = "AdminLib",
+    abi = "src/admin/out/debug/admin_test-abi.json"
+));
+
+pub struct Metadata {
+    pub contract: AdminLib<WalletUnlocked>,
+    pub wallet: WalletUnlocked,
+}
+
+pub mod abi_calls {
```

Comment:
```suggestion
pub mod interface {
```

Reply:
I am leaving this as is. It is the same structure followed in other tests across 3 repositories. `abi_calls` is clear, concise, and quickly understandable.

Reply:
That's fine.

I've pointed out that `interface` is what we generally use and is the canonical term for a reason.